system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -1614,7 +1617,10 @@ impl_runtime_apis! {
 				pub TrustedReserve: Option<(Location, Asset)> = Some(
 					(
 						xcm_config::bridging::to_polkadot::AssetHubPolkadot::get(),
-						Asset::from((xcm_config::bridging::to_polkadot::DotLocation::get(), 1000000000000 as u128))
+						Asset::from((
+							xcm_config::bridging::to_polkadot::DotLocation::get(),
+							1000000000000 as u128,
```

Comment:
```suggestion
							10000000000 as u128,
```

Not sure if that is so important for benchmarking? But DOT has 10 decimals.